Vagrantfile

```diff
@@ -50,8 +50,7 @@ Vagrant.configure("2") do |config|
     chef(config, "docker", host_name)
   end
 
-# To configure a digital ocean provider, uncomment the following
-=begin
```

Comment:
Can you comment the DO stuff back out please? Users will probably want to try the box out locally before hooking it up to DO.